tests/test_sqlite.py

```diff
@@ -1,10 +1,12 @@
 SECRET_KEY = "django_tests_secret_key"
-
 CACHES = {
     "default": {
         "BACKEND": "django_redis.cache.RedisCache",
         "LOCATION": ["redis://127.0.0.1:6379?db=1", "redis://127.0.0.1:6379?db=1"],
-        "OPTIONS": {"CLIENT_CLASS": "django_redis.client.DefaultClient"},
+        "OPTIONS": {
```

Comment:
Why are you modifying the settings for all the tests?

Reply:
This is a mistake, i restore

Reply:
I already shared [here](https://github.com/jazzband/django-redis/pull/492#pullrequestreview-567203974)

Reply:
If you mean `with self.settings(LOGIN_URL='/other/login/'):` it only works with django.tests.TestCase, not unittest.TestCase and django.tests.TestCase requires a database.

Reply:
You can use, `SimpleTestCase` (have a look [here](https://docs.djangoproject.com/en/3.1/topics/testing/tools/#simpletestcase)) like it is done in `django` (have a look [here](class DummyCacheTests(SimpleTestCase):))

Reply:
> You can use, `SimpleTestCase` (have a look [here](https://docs.djangoproject.com/en/3.1/topics/testing/tools/#simpletestcase)) like it is done in `django` (have a look [here](https://github.com/django/django/blob/83fcfc9ec8610540948815e127101f1206562ead/tests/cache/tests.py#L82))

My classes would be the only ones that extend SimpleTestCase. I think my test classes should stick to unittest.TestCase like the other ones or all the test classes should extend djange.test.SimpleTestCase

Reply:
You can do with plain `unittest.TestCase` or switch everything to `django.test.SimpleTestCase`, I have a plan to do some refactoring on the tests anyway

Reply:
I agree, the tests should be refactored.